docs/data/trigger-sync.md

```diff
@@ -1,31 +1,43 @@
 ---
 title: "Trigger Cloud Sync Conditionally"
 linkTitle: "Trigger Sync"
-description: "Trigger Cloud Sync to sync captured data conditionally."
+description: "Trigger cloud sync to sync captured data conditionally."
 weight: 20
 type: "docs"
 tags: ["data management", "cloud", "sync"]
 # SME: Alexa Greenberg
 ---
 
-If you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
+You can use a {{< glossary_tooltip term_id="module" text="module" >}} to sync data only when a certain logic condition is met, instead of at a regular time interval.
+For example, if you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
+Or, you may want to trigger sync only when your machine detects an object of a certain color.
+The code for both of these examples is provided by the [trigger-sync-examples module](https://github.com/viam-labs/trigger-sync-examples-v2), and you can [create your own module](/registry/create/) if you want to use different logic.
```

Comment:
Both of which examples? reference unclear.

Reply:
Does this work?
```suggestion
The code for the timeframe and color detection trigger examples is provided by the [trigger-sync-examples module](https://github.com/viam-labs/trigger-sync-examples-v2), and you can [create your own module](/registry/create/) if you want to use different logic.
```

Reply:
I am only just realizing that this maps to the first two sentences here which are about time frame and color detection example. I was somehow seeing that as disconnected... The original copy is fine actually - sorry!

Reply:
Okay great :)